examples/src/main/java/com/yugabyte/examples/TopologyAwareLoadBalanceExample.java

```diff
@@ -10,219 +10,21 @@
 import java.sql.*;
 import java.util.*;
 
-public class TopologyAwareLoadBalanceExample {
-  private static String controlUrlWithPlacement;
-  private static HikariDataSource hikariDataSource;
-  static Scanner scanner = new Scanner(System.in);
-  static List<Connection> borrowConnections = new ArrayList<>();
+public class TopologyAwareLoadBalanceExample extends UniformLoadBalanceExample{
 
   public static void main(String[] args) {
-    try {
+    Boolean VERBOSE = (args[0].equals("1") ? true : false);
```

Comment:
Why not change the case here also. Any specific reason to keep them uppercase?

Reply:
nothing, missed the changes in this class while refactoring the code.